packages/flutter/lib/src/services/text_input.dart

```diff
@@ -658,9 +658,11 @@ class TextEditingValue {
     this.text = '',
     this.selection = const TextSelection.collapsed(offset: -1),
     this.composing = TextRange.empty,
+    this.scribbleInProgress = false,
```

Comment:
Well, the reason I put it here is so that it can be used to set the selection changed cause appropriately here: https://github.com/twinsunllc/flutter/pull/1/files#diff-f5f93c879cef9a102adbf148583de9f3d9a05f9678fdcad7bde106f62200bb34R1663